denguefever_tw/dengue_linebot/utils.py

```diff
@@ -63,22 +63,35 @@ def push_msg(line_bot_api, users, text, img):
         logger.info('Fail to push message!')
 
 
-def get_web_screenshot(zapper_id):
+def get_web_info(zapper_id, mode):
+    web_info = dict()
+    if mode == 'area':
+        zapper_api = urljoin(BASE_ZAPPER_API_URL, 'lamps/{id}?key=hash'.format(id=zapper_id))
+        response = requests.get(zapper_api)
+        response_json = response.json()
+
+        params = urlencode({'lng': response_json['lamp_location'][0], 'lat': response_json['lamp_location'][1]})
+        web_info['url'] = urljoin(BASE_ZAPPER_API_URL, '/zapperTown/index.html?%s' % params)
+        web_info['width'] = 1200
+        web_info['height'] = 900
+    elif mode == 'self':
+        web_info['url'] = urljoin(BASE_ZAPPER_API_URL, '/zapperCitizen?%s' % zapper_id)
+        web_info['width'] = 1000
+        web_info['height'] = 700
+    else:
+        pass
```

Comment:
Why is this pass even needed?

Reply:
I use `pass` to do nothing because there are only two modes.
Is it not so good?